rocksdb_admin/admin_handler.cpp

```diff
@@ -238,6 +241,8 @@ AdminHandler::AdminHandler(
   if (db_manager_ == nullptr) {
     db_manager_ = CreateDBBasedOnConfig(rocksdb_options_);
   }
+  folly::split(",", FLAGS_allow_overlapping_keys_segments,
```

Comment:
Can we make allow_overlapping_keys_segments_  a set/unordered_set and use folly::splitTo()?
Though we don't really care about performance here, the code for looking up will be more elegant.